src/bots/ping/handler.js

```diff
@@ -0,0 +1,8 @@
+import client from '../../client';
+
+const handler = (_, msg) => {
```

Comment:
I'm not really a fan of just assuming all messages sent to a ping bot are pings.  Maybe this should be separated into ping and throw otherwise.